Bootstrap/SelectorTitleView/SelectorTitleView.swift

```diff
@@ -30,6 +30,9 @@ public class SelectorTitleView: UIView {
         }
     }
 
+    public var arrowUpImage: UIImage?
+    public var arrowDownImage: UIImage?
```

Comment:
Probably the button image needs to be updated when these are set too? Or make it possible to set them in init.

Reply:
I think init is the way to go, I'll update it.